ext/ckeditor4/CRM/Ckeditor4/Upgrader.php

```diff
@@ -0,0 +1,141 @@
+<?php
+use CRM_Ckeditor4_ExtensionUtil as E;
+
+/**
+ * Collection of upgrade steps.
+ */
+class CRM_Ckeditor4_Upgrader extends CRM_Ckeditor4_Upgrader_Base {
+
+  // By convention, functions that look like "function upgrade_NNNN()" are
+  // upgrade tasks. They are executed in order (like Drupal's hook_update_N).
+
+  /**
+   * Example: Run an external SQL script when the module is installed.
+   */
+  public function install() {
+    CRM_Core_BAO_OptionValue::ensureOptionValueExists([
+      'option_group_id' => 'wysiwyg_editor',
+      'label' => 'CKEditor',
```

Comment:
```suggestion
      'label' => E::ts('CKEditor 4'),
```

I think it would be good to update this label for existing installs as well. So either an additional call to update the label if it already exists, or if you want to be clever maybe you could do the create or update in a single call using the api3's `'options.match'`.